modules/meeting/app/workers/meetings/pdf/participants.rb

```diff
@@ -57,11 +56,24 @@ def participants_table_column_widths(columns_count)
     end
 
     def participants
-      meeting.invited_or_attended_participants
+      meeting.invited_or_attended_participants.sort_by(&:name)
+    end
+
+    def participants_groups(columns_count)
+      # note participants.in_groups does not work with the alphabetically sorted requirement
+      # should be left to right and then the next row
+      array = Array.new(columns_count) { [] }
+      chunks = participants.in_groups_of(columns_count)
+      chunks.each do |chunk|
+        chunk.each_with_index do |participant, participant_index|
+          array[participant_index] << participant
+        end
+      end
+      array
```

Comment:
This method could be shortened to:

```ruby
participants.each_slice(columns_count).to_a.transpose
```

However, not many people know `transpose`, so it might be better to keep the current, more explicit version.

Reply:
@EinLama And I am one of those not knowing :D I will keep it in mind, note this occurrence down and change it in another probable iteration of this export. Thanks!